lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -240,10 +242,10 @@ defmodule ExUnit.Callbacks do
   @doc """
   Defines a callback to be run before all tests in a case.
 
-  Pass a block or name of a unary function as atom, or list of such
-  atoms.
+  Accepts a block or the name of a publicly accessible one-arity function in the shape of an atom,
```

Comment:
Do they have to be publicly accessible? I don't think they do.

Reply:
I will check again.
I faced this problem some time ago and I am sure the setup was not working because this function was private.

Reply:
You are right!